frontend/src/components/mine/Compliance/ComplianceOrdersTable.js

```diff
@@ -19,6 +19,20 @@ const defaultProps = {
   filteredOrders: [],
 };
 
+const codeSorter = (code_one, code_two) => {
+  // if a code is null it set before one that is not in sort order.
+  if (!code_one) {
+    return true;
+  }
+  if (!code_two) {
+    return false;
+  }
+  if (compareCodes(code_one, code_two) < 0) {
+    return true;
+  }
+  return false;
```

Comment:
These final returns are redundant due to the conditional. So, if the conditional passes (is true), you're returning true, otherwise you're returning false. In both cases, you return the value of the condition.

So, this will do the same thing:
```javascript
return compareCodes(code_one, code_two) < 0;
```

Also, I'd like to push for an explicit null check instead of the bang checks for the presence checks.

And I'm sure it was just a habit carried over from the python code, but JS favours camelCase, so we should stick to that when possible.